lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -9,10 +9,16 @@ triggers startend => 'quote', 'quotes', 'quotation', 'quotations';
 spice to => 'https://www.brainyquote.com/api/ddg?q=$1';
 spice wrap_jsonp_callback => 1;
 
+spice alt_to => {
```

Comment:
Can you please clarify further?  When I call $.getJSON("/ddg/spice/brainy_quote/" +  query) I receive an object with HTML response text.